index.html

```diff
@@ -0,0 +1,192 @@
+                    <img src="./imgs/icons/team.png" alt="team icon">
+                    <h5 class="title text-main">Team members</h5>
+                    <p class="desc ">Individuals collaborating to achieve common goals, leveraging diverse skills
+                        and expertise.
+                    </p>
+                </div>
+                <div class="align-items-center column-gap-4 d-flex flex-md-column p-3 prog row-gap-md-2">
+                    <img src="./imgs/icons/goal.png" alt="goal icon">
+                    <h5 class="title text-main">Winning awards</h5>
+                    <p class="desc ">in youth leadership celebrates the exceptional abilities and contributions of
+                        young leaders.
+                    </p>
+                </div>
+                <div class="align-items-center column-gap-4 d-flex flex-md-column p-3 prog row-gap-md-2">
+                    <img src="./imgs/icons/arm-wrestling.png" alt="arm arm-wrestling icon">
+                    <h5 class="title text-main">Training programmes</h5>
+                    <p class="desc ">Structured learning initiatives to enhance knowledge, skills, and competencies
+                        in specific areas.
+                    </p>
+                </div>
+                <div class="align-items-center column-gap-4 d-flex flex-md-column p-3 prog row-gap-md-2">
+                    <img src="./imgs/icons/list.png" alt="list icon">
+                    <h5 class="title text-main">Completed project</h5>
+                    <p class="desc ">Successfully executed initiatives that meet objectives within time and budget
+                        constraints.
+                    </p>
+                </div>
+                <div class="align-items-center column-gap-4 d-flex flex-md-column p-3 prog row-gap-md-2">
+                    <img src="./imgs/icons/success.png" alt="success icon">
+                    <h5 class="title text-main">Leadership Development</h5>
+                    <p class="desc ">Initiatives nurturing leaders' skills to inspire and guide others toward
+                        organizational goals.
+                    </p>
+                </div>
+            </div>
+
+            <div class="btn-container d-flex justify-content-center py-4 mt-4"><button class="d-md-none"
+                    type="button">Join AYSDN
+                    now</button>
+                <h3 class="text-center d-md-block d-none"><a href="#" class="seeAll">see all interests area</a>
+                </h3>
+            </div>
+        </section>
+
+        <section class=" team-members container" id="Members">
+            <h2 class="text-center text-dark pt-5">Team members</h2>
+            <div class="line"></div>
+
+            <div class="teams mt-5 d-md-flex flex-md-column px-md-5 px-lg-5">
+                <div class="row" id="membersContainer">
+
+                </div>
```

Comment:
- [ ] [OPTIONAL] Please, it could be better if you add `more` and `less` buttons to display `members` in the mobile version 😃. 

![Screenshot 2023-05-23 152755](https://github.com/Mofuhidy/first-capstone-project/assets/57408419/fc4100ee-d395-4706-86c4-23fd3351c66b)